setup.py

```diff
@@ -77,8 +82,20 @@
     keywords="fileformat nif cgf binary interface stripify",
     platforms=["any"],
     description="Processing block structured binary files.",
-    classifiers=[_f for _f in classifiers.split("\n") if _f],
+    classifiers=classifiers,
     long_description=long_description,
     url="https://github.com/niftools/pyffi",
-    download_url="https://github.com/niftools/pyffi/releases"
+    download_url="https://github.com/niftools/pyffi/releases",
+    cmdclass={'build_docs': BuildDoc},
```

Comment:
Should we not alias this in setup.cfg

Reply:
We could in fact move a lot of things inside setup.py into setup.cfg to clean up the setup.py script.
`classifiers`, `packages`, `package_data` and `scripts` can be put into `setup.cfg` to save on many lines

Reply:
Actually was more about creating an actual alias for python setup.py docs. Not sure of what the standard operations/conventions are for setup.py.

@psi29a You're probably more in tune with this as you mapped the test operation to abstact away what underlying test framework we actually use -
 https://github.com/TagnumElite/pyffi/blob/80ae40191766622795d36384fac66c020ac00719/.travis.yml#L26

Also we should also consider moving the coverage stuff to setup.cfg as per the discussion around but out of scope of current

Reply:
I have no problem pushing more things out to setup.cfg, I had no idea the cmdclass trick existed so colour me impressed. You're right, we shouldn't require sphinx in order to use pyffi.

Reply:
Yeah I was just wondering if there were certain standard command, eg in java land you have maven which has compile, test, install etc. was wondering if there were was similiar in setup.py. Probably should apply some google-fu at this point.